pkg/crane/push.go

```diff
@@ -43,6 +43,16 @@ func LoadTag(path, tag string, opt ...Option) (v1.Image, error) {
 	return tarball.ImageFromPath(path, &t)
 }
 
+// Put writes the remote.Taggable to a registry as dst.
+func Put(put remote.Taggable, dst string, opt ...Option) error {
```

Comment:
Is `dst` expected to be a repo, tag or digest? Can we make that clearer from the comment and/or validation and error messages?